pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.sagebionetworks.bridge</groupId>
     <artifactId>sdk-group</artifactId>
     <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-    <version>0.16.27</version>
+    <version>0.16.28</version>
```

Comment:
Note that this conflicts with https://github.com/Sage-Bionetworks/BridgeJavaSDK/pull/543

Reply:
This will get merged first, so I will update the versions of the other PRs.